Modix.Services/Core/RolesLogMessages.cs

```diff
@@ -0,0 +1,135 @@
+                logger);
+        private static readonly Action<ILogger> _roleCreating
+            = LoggerMessage.Define(
+                    LogLevel.Debug,
+                    EventType.RoleCreating.ToEventId(),
+                    "Creating database record for role")
+                .WithoutException();
+
+        public static void RoleTracked(
+                ILogger logger)
+            => _roleTracked.Invoke(
+                logger);
+        private static readonly Action<ILogger> _roleTracked
+            = LoggerMessage.Define(
+                    LogLevel.Debug,
+                    EventType.RoleTracked.ToEventId(),
+                    "Role tracked within database")
+                .WithoutException();
+
+        public static void RoleTracking(
+                ILogger logger)
+            => _roleTracking.Invoke(
+                logger);
+        private static readonly Action<ILogger> _roleTracking
+            = LoggerMessage.Define(
+                    LogLevel.Debug,
+                    EventType.RoleTracking.ToEventId(),
+                    "Tracking role within database")
+                .WithoutException();
+
+        public static void RoleUpdated(
+                ILogger logger)
+            => _roleUpdated.Invoke(
+                logger);
+        private static readonly Action<ILogger> _roleUpdated
+            = LoggerMessage.Define(
+                    LogLevel.Debug,
+                    EventType.RoleUpdated.ToEventId(),
+                    "Database record updated for role")
+                .WithoutException();
+
+        public static void RoleUpdateFailed(
+                ILogger logger)
+            => _roleUpdateFailed.Invoke(
+                logger);
+        private static readonly Action<ILogger> _roleUpdateFailed
+            = LoggerMessage.Define(
+                    LogLevel.Debug,
+                    EventType.RoleUpdateFailed.ToEventId(),
+                    "Database record for role not found")
+                .WithoutException();
+
+        public static void RoleUpdating(
+                ILogger logger)
+            => _roleUpdating.Invoke(
+                logger);
+        private static readonly Action<ILogger> _roleUpdating
+            = LoggerMessage.Define(
+                    LogLevel.Debug,
+                    EventType.RoleUpdating.ToEventId(),
+                    "Updating database record for role")
+                .WithoutException();
+
+        public static void TransactionBeginning(
+                ILogger logger)
+            => _transactionBeginning.Invoke(
+                logger);
+        private static readonly Action<ILogger> _transactionBeginning
+            = LoggerMessage.Define(
+                    LogLevel.Debug,
+                    EventType.TransactionBeginning.ToEventId(),
+                    "Beginning database transaction")
+                .WithoutException();
+
+        public static void TransactionComitting(
```

Comment:
Minor, typo: "Comitting" -> "Committing"

Reply:
So, why didn't you suggest the change? GOSH!

Reply:
Because it requires changes in multiple places and I didn't feel like commenting on all of them 😛